### DISCUSSION TITLE
A new import/export blocks idea

### DISCUSSION THREAD

**Discussion by Tomtfc** [TurboWarp/packager#249] [Ideas]
A block like this opens a file browser to import a file to the selected list

Import data to list (listname) file extension (file extension)

Listname - the list Which the selected file will be imported to

File extension - the extension/extensions which the file browser shows

**Comment by hello-smile6** [comment]
> A block like this opens a file browser to import a file to the selected list
> 
> Import data to list (listname) file extension (file extension)
> 
> Listname - the list Which the selected file will be imported to
> 
> File extension - the extension/extensions which the file browser shows

You could maybe use the code from https://gitlab.com/adacraft for that. Also, wouldn't this be for scratch-gui or scratch-vm?
